services/speech_to_text/v1.js

```diff
@@ -85,20 +98,39 @@ module.exports = function (RED) {
 
       var s2t = function (audio, format, cb) {
         if (format !== 'wav' && format !== 'flac' && format !== 'ogg') {
-          var message = 'Audio format (' + format + ') not supported, must be encoded as WAV, FLAC or OGG.';
-          node.error(message, msg)
+          var message_err_format = 'Audio format (' + format + ') not supported, must be encoded as WAV, FLAC or OGG.';
+
+          node.error(message_err_format, msg)
           return;
         }
 
-        if (format === 'ogg') format += ';codecs=opus';
+        if (format === 'ogg') { 
+          format += ';codecs=opus';
+        }
+
+        var params = {};
+        
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Opening curly brace does not appear on the same line as controlling statement.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2399832750/issues/source?bid=3225115&fileBranchId=3231190#l113)